WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -32,34 +34,34 @@ struct CartView: View {
             .foregroundColor(Color.white)
             ScrollViewReader { proxy in
                 ScrollView {
-                    ForEach(viewModel.itemsInCart, id: \.id) { cartItem in
+                    ForEach(cartViewModel.itemsInCart, id: \.id) { cartItem in
                         ItemRowView(cartItem: cartItem,
-                                    onItemRemoveTapped: viewModel.canDeleteItemsFromCart ? {
-                            viewModel.removeItemFromCart(cartItem)
+                                    onItemRemoveTapped: cartViewModel.canDeleteItemsFromCart ? {
+                            cartViewModel.removeItemFromCart(cartItem)
                         } : nil)
                         .id(cartItem.id)
                         .background(Color.posBackgroundGreyi3)
                         .padding(.horizontal, 32)
                     }
                 }
-                .onChange(of: viewModel.itemToScrollToWhenCartUpdated?.id) { _ in
-                    if viewModel.orderStage == .building,
-                       let last = viewModel.itemToScrollToWhenCartUpdated?.id {
+                .onChange(of: cartViewModel.itemToScrollToWhenCartUpdated?.id) { _ in
+                    if cartViewModel.orderStage == .building,
+                       let last = cartViewModel.itemToScrollToWhenCartUpdated?.id {
                         withAnimation {
                             proxy.scrollTo(last)
                         }
                     }
                 }
             }
             Spacer()
-            switch viewModel.orderStage {
+            switch cartViewModel.orderStage {
             case .building:
                 checkoutButton
                     .padding(32)
             case .finalizing:
                 addMoreButton
                     .padding(32)
-                    .disabled(viewModel.isAddMoreDisabled)
+                    .disabled(dashboardViewModel.isAddMoreDisabled)
```

Comment:
This `.disabled` state relies on the payment state, possibly shouldn't be attached to the cart logic, but totals.

Reply:
Are you planning to move forward with this PR @iamgabrielma?

I noticed it involves `isAddMoreDisabled` which I've updated in my [PR](https://github.com/woocommerce/woocommerce-ios/pull/13179) to also take into account order syncing state.

Reply:
Thanks for the ping! This can be closed as was just a preliminary draft for the following changes:
- https://github.com/woocommerce/woocommerce-ios/pull/13188
- https://github.com/woocommerce/woocommerce-ios/pull/13191
- https://github.com/woocommerce/woocommerce-ios/pull/13211